berkeley-mobile/Resources/Covid/CovidResourceViewController.swift

```diff
@@ -183,6 +183,65 @@ extension CovidResourceViewController {
+        let card = CardView()
+        card.layoutMargins = kCardPadding
+        
+        scrollingStack.stackView.addArrangedSubview(card)
+        
+        card.translatesAutoresizingMaskIntoConstraints = false
+        card.leftAnchor.constraint(equalTo: scrollingStack.layoutMarginsGuide.leftAnchor).isActive = true
+        card.rightAnchor.constraint(equalTo: scrollingStack.layoutMarginsGuide.rightAnchor).isActive = true
+        
+        let headerLabel = UILabel()
+        headerLabel.text = "Need to get tested?"
+        headerLabel.font = Font.medium(20)
+        headerLabel.textAlignment = .left
+        headerLabel.textColor = Color.blackText
+        headerLabel.numberOfLines = 1
+        headerLabel.adjustsFontSizeToFitWidth = true
+        headerLabel.minimumScaleFactor = 0.7
+        
+        let subtitleLabel = UILabel()
+        subtitleLabel.text = "Book an appointment directly through University Health Services:"
+        subtitleLabel.font = Font.medium(12)
+        subtitleLabel.numberOfLines = 2
+        subtitleLabel.textAlignment = .left
+        subtitleLabel.textColor = Color.blackText
+        
+        card.addSubview(headerLabel)
+        card.addSubview(subtitleLabel)
+        
+        headerLabel.translatesAutoresizingMaskIntoConstraints = false
+        headerLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
+        headerLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        headerLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+
+        subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        subtitleLabel.topAnchor.constraint(equalTo: headerLabel.bottomAnchor, constant: 8).isActive = true
+        subtitleLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        subtitleLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+        
+        let bookButton = UIButton()
+        bookButton.titleLabel?.font = Font.medium(16)
+        bookButton.setTitle("Book Now", for: .normal)
+        bookButton.setTitleColor(.white, for: .normal)
+        bookButton.backgroundColor = Color.lowOccupancyTag
+        bookButton.layer.cornerRadius = 16.5
+        bookButton.addTarget(self, action: #selector(bookButtonPressed), for: .touchUpInside)
+        
+        card.addSubview(bookButton)
+        
+        bookButton.translatesAutoresizingMaskIntoConstraints = false
+        bookButton.topAnchor.constraint(equalTo: subtitleLabel.bottomAnchor, constant: kViewMargin).isActive = true
+        bookButton.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        
+        bookButton.setHeightConstraint(33)
+        bookButton.setWidthConstraint(110)
+        
+        card.bottomAnchor.constraint(equalTo: bookButton.bottomAnchor, constant: kViewMargin).isActive = true
```

Comment:
Would have constraining the `bookButton.bottomAnchor.constraint(equalTo: card.layoutMarginsGuide.bottomAnchor).isActive = true` done the same thing? If so, I think we should use that instead since it uses `kCardPadding` at the top of this function.